test/wrapper.auto.tfvars

```diff
@@ -51,8 +52,12 @@ additional_node_pools = [
     minimum_node_count = 1
     maximum_node_count = 3
     enable_secure_boot = true
-    taints             = []
-    tags               = ["terratest"]
-    zones              = ["asia-southeast2-a", "asia-southeast2-b", "asia-southeast2-c"]
+    taints = [{
```

Comment:
You're no longer testing for zero taints being supplied. To test the module logic there should be a test case with taints, and one with no taints.

Reply:
Added a node pool for zero taints. CI is running.